source/Nevermore/AST/SelectConstant.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Nevermore.AST
+{
+    public class SelectConstant : ISelect 
+    {
+        readonly string constant;
+
+        public SelectConstant(string constant)
+        {
+            this.constant = constant;
+        }
+
+        public string GenerateSql()
+        {
+            return $"SELECT {constant}";
```

Comment:
What if the `constant` is a string? Also could this be a vector for an injection attack?